src/Components/Project.js

```diff
@@ -0,0 +1,177 @@
+import React, { useState } from "react";
+import Carousel from "react-bootstrap/Carousel";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import { Container } from "react-bootstrap";
+
+import logo from "../assets/images/hero1.jpg";
+import eee from "../assets/images/eee.png";
+import blackjack from "../assets/images/blackjack.png";
+
+function Project() {
+  const [index, setIndex] = useState(0);
+
+  const handleSelect = (selectedIndex, e) => {
+    setIndex(selectedIndex);
+  };
+
+  return (
+    <Container id="project">
+      <h1
+        style={{
+          textAlign: "center",
+          marginBottom: "2vh",
+          fontSize: "max(3vw, 22pt)",
+          fontFamily: "Rubik Mono One",
+          color: "#20C20E",
+        }}
+      >
+        5.Projects
+      </h1>
+      <Carousel
+        activeIndex={index}
+        onSelect={handleSelect}
+        variant="light"
+        className="centerDiv"
+        style={{
+          flexDirection: "column",
+          paddingBottom: "10vh",
+          color: "#20C20E",
+          fontFamily: "Share Tech Mono",
+        }}
+      >
+        <Carousel.Item>
```

Comment:
The carousel item code is very repetitive, and therefore we can use a .map to render everything nicely, without much manual work.